.github/workflows/publish.yml

```
@@ -18,39 +18,56 @@ jobs:
-    name: Deploy
+  publish:
+    name: Publish
     needs: prepare
     runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@master
-
-      - name: Setup Node
-        uses: ./.github/actions/setup-node
+    runs-on: ${{ matrix.os }}
```

actionlint (1.7.7)

key "runs-on" is duplicated in "publish" job. previously defined at line:24,col:5

(syntax-check)